sql.txt

```diff
@@ -1,14 +1,45 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName,
+        ContactName,
+        Country
+FROM Customers
+ORDER BY country
+LIMIT 20
+OFFSET 10;
```

Comment:
The assignment asked for entries 11 through 20, but this loads more entries than that. Could you update this so that only the entries that the assignment asks for show up?